assets/js/src/bulk-delete.js

```diff
@@ -13,6 +13,37 @@ jQuery(document).ready(function () {
 	 */
 	jQuery( '.select2' ).select2();
 
+	jQuery( '.select2Ajax' ).select2({
+		ajax: {
+    			url: ajaxurl, 
+    			dataType: 'json',
+    			delay: 250, 
+    			data: function (params) {
+    				var term = jQuery(this).attr('data-term');
+      				return {
+        				q: params.term, 
+        				term: term,
```

Comment:
This is actually the taxonomy. So change the variable and object key to `taxonomy` to make it more readable.